2021/_includes/homepage/sponsors.html

```diff
@@ -18,13 +18,21 @@ <h3>Sponsors<span class="separator">
                 <div class="sponsor-top">
                     <h1>Top</h1>
                     <br>
-                    <br>
-                    <a href="{{ site.data.sponsors.top.href }}" target="_blank" title="{{ site.data.sponsors.top.name }}">
-                        <img src="{{base}}{{ site.data.sponsors.top.image.src }}" style="max-height:100%; max-width:100%"
-                             class="attachment-eventstation-related-post-image size-eventstation-related-post-image wp-post-image"
-                             alt="{{ site.data.sponsors.top.image.alt }}"/>
-                    </a>
-                    <br>
+                    <p>
+                        <a href="{{ site.data.sponsors.top[0].href }}" target="_blank" title="{{ site.data.sponsors.top[0].name }}">
```

Comment:
```suggestion
                        <a href="{{ site.data.sponsors.top[0].href }}" target="_blank" title="{{ site.data.sponsors.top[0].name }}" rel=”noreferrer”>
```

Reply:
Not sure if I got your suggestion: if our Sponsors are getting visits because they come from our website, why we should avoid that? 🤔

Reply:
this is the description from the [Github Code scanning tool](https://github.com/barcelonajug/jbcnconf_web/security/code-scanning/344?query=ref%3Arefs%2Fpull%2F967%2Fmerge):

>  External links without noopener/noreferrer are a potential security risk.

We can also dismiss it if you think it is not relevant